src/script/util/DebugUtil.ts

```diff
@@ -0,0 +1,319 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```

Comment:
since it's a new file maybe 2020?

Reply:
It's not really a new file but a renamed one. Copyright on the code still stays the same.